projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/license.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+import { __, _x } from '@wordpress/i18n';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import TextInput from 'components/text-input';
+import { createNotice } from 'components/global-notices/state/notices/actions';
+
+class License extends Component {
```

Comment:
We try to stay away from classes for new components, and now use function components instead:
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/_inc/client/README.md#L8-L10